include/psp2/message_dialog.h

```diff
@@ -38,6 +38,7 @@ typedef enum SceMsgDialogSystemMessageType {
 	SCE_MSG_DIALOG_SYSMSG_TYPE_NOSPACE                        = 2,
 	//! Displays "Move away from the source of interference, or adjust the compass by moving your PS Vita system as shown below."
 	SCE_MSG_DIALOG_SYSMSG_TYPE_MAGNETIC_CALIBRATION           = 3,
+	SCE_MSG_DIALOG_SYSMSG_TYPE_MIC_DISABLED                   = 4,
```

Comment:
Can you add message displayed for this? Same as the other messages in the enum

Reply:
I removed the enums as some messages can be quite long and I do not know how to format them to fit with what is expected here. I will let someone else handle it.